index.html

```diff
@@ -0,0 +1,209 @@
+                </p>
+                <h2 class="a">LET’S CONNECT</h2>
+
+                   <ul class="social-media">
+                       <a href="##"><li> <img src="./image/twit.png" alt=""></li></a> 
+                       <a href="##"><li><img src="./image/linked.png" alt=""></li></a> 
+                       <a href="##"><li><img src="./image/mail.png" alt=""></li></a> 
+                       <a href="##"> <li><img src="./image/pp.png" alt=""></li></a> 
+                       <a href="##"><li><img src="./image/peace.png" alt=""></li></a> 
+                   </ul>                 
+               </section>
+        
+   </div>
+        <section>
+            <div class="card1">
+            <img src="./image/image2.png" alt="hello">
+        <h1>TONIC</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+    </div>
+
+        <div class="card2">
+            <img src="./image/card2.png" alt="some">
+        <h1>Multi-Post <br>Stories</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+         </div>
+
+
+        <div class="card3">
+            <img src="./image/card3.png" alt="">
+        <h1>TONIC</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+            
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>
+
+
+
+
+
+
+
+
+        <div class="card4">
+            <img src="./image/card4.png" alt="">
+        <h1>Multi-Post <br>Stories</h1> 
```

Comment:
Kindly, I suggest you only put an H1 heading, for other titles follow the sequence H2, H3... depending on the importance. this will greatly help the accessibility of your website.